ls1intum/Artemis
# `Plagiarism checks`: Show an alert to the instructor when the plagiarism notification case has been sent

### Checklist
#### General



#### Client

### Motivation and Context


Closes #5520. When the instructor notifies a student about a plagiarism case, they don't receive any message if the notification has been sent. 
### Description

Show an alert to the instructor when the plagiarism notification case has been sent to the student.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exercise with Plagiarism Case

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to the Plagiarism Case
4. Notify the student and see that a success alert is shown

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
| plagiarism-case-instructor-detail-view.component.ts | 82.35% | 98.03% |

### Screenshots



![image](https://user-images.githubusercontent.com/18427209/182447515-b68bb0c4-af19-40c3-b058-e98d330aa217.png)

![image](https://user-images.githubusercontent.com/18427209/182447585-8e83c2f5-f5ab-468b-9d1d-dbcc366e3bb0.png)
